docs/components/modules/ROOT/pages/any23-dataformat.adoc

```diff
@@ -0,0 +1,119 @@
+[[any23-dataformat]]
+== Any23 DataFormat
+== TidyMarkup DataFormat
```

Comment:
Please change the docs and talk about Any23 project, this part is static so you need to write some sentences in the .adoc